### PR TITLE
Copy dispatch ui to release ova

### PR DESCRIPTION
**Pull request by pzmrzy**

### CODE REVIEW

File: ci/pipelines/ova-release.yml [hunk 1]
```diff
@@ -98,6 +104,14 @@ jobs:
       params:
         file: dispatch-logs/*.tar.gz
 
+- name: build-dispath-ui
+  public: true
+  plan:
+  - get: dispatch-ui
+    resource: dispatch-ui
+  - task: build-ui
+    file: dispatch-ui/ci/tasks/build-bundle.yml
```
Comment (berndtj) [inline ci/pipelines/ova-release.yml]: Is this file missing from the commit?

Comment (pzmrzy) [reply ci/pipelines/ova-release.yml]: No, it located in ui repo
https://github.com/dispatchframework/dispatch-ui/blob/master/ci/tasks/build-bundle.yml

File: ci/pipelines/ova-release.yml [hunk 2]
```diff
@@ -98,6 +104,14 @@ jobs:
       params:
         file: dispatch-logs/*.tar.gz
 
+- name: build-dispath-ui
```
Comment (kars7e) [inline ci/pipelines/ova-release.yml]: s/dispath/dispatch/g

File: ci/pipelines/ova-release.yml [hunk 3]
```diff
@@ -108,10 +122,15 @@ jobs:
   - get: dispatch
     resource: dispatch-solo
     passed: [test-dispatch-solo]
+  - get: dispatch-ui
+    resource:  dispatch-ui
+    passed: [build-dispath-ui]
```
Comment (kars7e) [inline ci/pipelines/ova-release.yml]: s/dispath/dispatch/g

File: ci/tasks/build-ova.yml [hunk 4]
```diff
@@ -10,6 +10,9 @@ inputs:
 - name: dispatch
   path: src/github.com/vmware/dispatch
 
+- name: dispatch-ui-binaries
+  path: src/github.com/vmware/dispatch-ui-binaries
```
Comment (kars7e) [inline ci/tasks/build-ova.yml]: this should be optional, now ova-build tests will fail

Comment (pzmrzy) [reply ci/tasks/build-ova.yml]: can make the file in `ova-manifest.json` optional?

**Pull request merged**